src/store/actions/bath.js

```diff
@@ -1,15 +1,48 @@
-// import {
-//   createSession
-// } from './index.js';
-//
-// import API_BASE_URL from '../../resources/API';
-//
-//
-//
-// export const editReview = (rating, text, restaurantId, reviewId, token) => {
-//   const url = API_BASE_URL + '/restaurants/' + restaurantId + '/review/' + reviewId;
+import {
+  setSentences,
+  saveSuccess,
+} from './index.js';
+
+import API_BASE_URL from '../../resources/API_URL';
+import DEFAULT_USER from '../../resources/defaultUser';
+
+
+export const runBath = (user = DEFAULT_USER) => {
+  const url = `${API_BASE_URL}/user/${user}/lexemes/10`;
+  const params = { method: 'GET' }
+
+  return fetch(url, params)
+    .then( response => response.json() )
+    .then( sentences => setSentences( sentences ) )
+    .catch( error => { } );
+}
+
+export const reportSuccess = (lexemeId, success, wrongAnswer, user = DEFAULT_USER) => {
+  const url = `${API_BASE_URL}/user/${user}/lexemes/${lexemeId}/${success?'correct':'wrong'}`;
+  const params = { method: 'PUT' }
+
+  return fetch(url, params)
+    .then( response => {
```

Comment:
This could be refactored with a ternary operator:
```
.then( response => response.ok 
   ? saveSucces()
   : {message: 'error' + response.status}
)

```